tests/test_class_loading.py

```diff
@@ -137,3 +144,70 @@ def test_find_test_module_of_class(mock_index):
     path = index.find_test_module_of_class("TestFixture")
     expected = "tests.base_tests.class_loading"
     assert path == expected
+
+
+##
+# Cloudy side tests
+
```

Comment:
I like the annotation, but like the comment "sunny side tests", comments should not explain what code does (code smell), but why - and the test methods are very indicative of what is going on :) so I'd delete those comments

Reply:
Ok, agreed. I removed the comments again.